Added release notes templates and package indexes for Android and iOS. (#1899)
Azure/azure-sdk: releases/2020-10/ios.md

---
title: Azure SDK for iOS (October 2020)
layout: post
tags: ios azure
sidebar: releases_sidebar
repository: azure/azure-sdk-for-ios
---

> The Azure SDK team is pleased to make available the October 2020 client library release.

Reviewer: ```suggestion
The Azure SDK team is pleased to make available the October 2020 client library releases.
```

#### GA

- _Add packages_

#### Updates

- _Add packages_

> #### Beta

Author: ```suggestion
#### Preview
```

- _Add packages_

## Installation Instructions

> To install the latest GA and beta libraries, use the [Swift Package Manager](https://swift.org/package-manager/). If you are using a different tool, refer to its documentation on how to specify dependencies.

Reviewer: ```suggestion
To install the latest GA and beta libraries, use the [Swift Package Manager](https://swift.org/package-manager/).
```

Author: @bsiegel I guess this means we expect people to only use Swift Package Manager, correct?

Reviewer: SwiftPM is the only package manager that is supported by the SDK for iOS.

> To install the latest GA and beta libraries, use the [Swift Package Manager](https://swift.org/package-manager/). If you are using a different tool, refer to its documentation on how to specify dependencies.

Author: ```suggestion
To install the latest GA and preview libraries, use the [Swift Package Manager](https://swift.org/package-manager/). If you are using a different tool, refer to its documentation on how to specify dependencies.
```

### Swift CLI

Follow the example in [Importing Dependencies](https://swift.org/package-manager/#importing-dependencies):

Open your project's `Package.swift` file and add a new package dependency to your project's `dependencies` section, specifying the clone URL of the repository and the version specifier you wish to use:

```swift
// Insert dependencies here
```

Next, add each client library you wish to use in a target to the target's array of `dependencies`:

```swift
// Insert dependencies here
```

## Feedback

If you have a bug or feature request for one of the libraries, please post an issue to [GitHub](https://github.com/azure/azure-sdk-for-ios/issues).

## Changelog

Detailed changelogs are linked from the [Quick Links](#quick-links) below. Here are some of the highlights:

### _Package name_

- Major changes only!

> ## Latest Releases

Reviewer: ```suggestion
## Need help
```

- For reference documentation visit the [Azure SDK for iOS documentation](https://azure.github.io/azure-sdk-for-ios/).

> - For tutorials, samples, quick starts and other documentation, visit [Azure for Mobile Developers](https://github.com/azure/azure-sdk-for-ios/).

Reviewer: ```suggestion
- For tutorials, samples, quick starts and other documentation, visit the [Azure SDK for iOS repository](https://github.com/azure/azure-sdk-for-ios/).
```

- File an issue via [Github Issues](https://github.com/Azure/azure-sdk-for-ios/issues/new/choose).

> - Check [previous questions](https://stackoverflow.com/questions/tagged/azure-ios-sdk) or ask new ones on

Author: @bsiegel The comment below implies we won't use the tag specified in this link, so we might want to change it, unless we plan on using the new tag as well.

Reviewer: Oh, yeah I didn't see that one. One sec.

> - Check [previous questions](https://stackoverflow.com/questions/tagged/azure-ios-sdk) or ask new ones on

Reviewer: ```suggestion
- Check [previous questions](https://stackoverflow.com/questions/tagged/azure+ios) or ask new ones on
```

>  StackOverflow using the `azure-ios-sdk` tag.

Reviewer: ```suggestion
 StackOverflow using the `azure` and `ios` tags.
```

Author: Would it be a good idea to create a new tag for this in addition to those two?

Reviewer: Probably, but this is what's in our README and is how users were told previously to tag questions about Colby's libraries. If we change this in the future we should do it in both places, and maybe see if we can migrate or back-tag those issues.

## Latest Releases

> View all the latest versions of iOS packages [here][android-latest-releases].

Reviewer: ```suggestion
View all the latest versions of iOS packages [here][ios-latest-releases].
```
